### PR TITLE
updated large promo for resizer

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Description

Update Large Promo to use image resizer

## Jira Ticket

- [TMEDIA-939](https://arcpublishing.atlassian.net/browse/TMEDIA-TMEDIA-939)

## Acceptance Criteria

- Large Promo block is using image resizer v2

## Test Steps

1. Checkout this branch `git checkout TMEDIA-939-large-promo-resizer`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/large-promo-block`
3. Add the large promo block to a page and configure the content source _id - CTAR7XCGLBCAXCHLJ6D5CZEGUA
4. Verify the Image is now using resizer v2 to display image
5. Test the integrated photo search to verify images selected from the integrated photo search also using resizer v2

## Author Checklist

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/large-promo-block/features/large-promo/default.jsx [hunk 1]
```diff
@@ -316,9 +317,27 @@ const LargePromoItem = ({ customFields, arcSite }) => {
...
+	const promoImageParams =
+		showImage &&
+		(imageOverrideURL || content
+			? {
+					src: imageOverrideURL
+						? {
+								_id: imageOverrideId,
+								url: imageOverrideURL,
+								auth: imageOverrideAuth ? JSON.parse(imageOverrideAuth) : null,
+						  }.url
```
Comment (nschubach) [inline blocks/large-promo-block/features/large-promo/default.jsx]: Why do we build up this object and only grab the `url` out of it?

File: blocks/large-promo-block/features/large-promo/default.jsx [hunk 2]
```diff
@@ -316,9 +317,27 @@ const LargePromoItem = ({ customFields, arcSite }) => {
...
+						: getImageFromANS(content).url,
+					alt: content?.headlines?.basic || "",
+					resizedOptions: {
```
Comment (nschubach) [inline blocks/large-promo-block/features/large-promo/default.jsx:334]: `auth` should be passed in here

**Review by nschubach — CHANGES_REQUESTED** [review]
two concerns

File: blocks/large-promo-block/features/large-promo/default.jsx (hunk 2 above)
Comment (matthewroach) [reply blocks/large-promo-block/features/large-promo/default.jsx:334]: Auth should be within the `ans` content now. @malavikakoppula, align the implementation with that of the extra large promo

**Pull request closed without merging**